docs/community-standars.md

```diff
@@ -0,0 +1,46 @@
+
+- **English is the primary working language of Openbullet, but large portions of the community are not native English speakers**. Be patient when language issues arise, and do not mistake incomplete language knowledge for ignorance or worse. If you are having trouble communicating an idea in English, please post in your native language and ask for help translating, and someone is likely to understand.
+
+- Openbullet is created exclusively by volunteers, as set out in our Mission Statement. Since they are freely giving their time and effort, **no volunteer contributor owes anything whatsoever to any other contributor, any user, or the project itself**. Contributors are free to come and go as they please, to work on and give attention to what they deem interesting or important, and to respond or not respond to anything they wish. **Do not badger volunteers** in any way about any topic.
+
+- If you have questions to ask of the community, **please choose the appropriate location** (see the Openbullet [forum](https://discourse.openbullet.dev/)) and **ask your question in full, immediately, with as much detail as possible**. Author issues aside, [Asking Questions the Smart Way](http://www.catb.org/~esr/faqs/smart-questions.html) is a valuable resource and is recommended reading before engaging with any free and open source community, including ours. **Do not pester or harass** community members for help or to answer questions. **Do not require others to pry information** out of you. **Do not spam questions**; they will be answered when they are answered.
+
+## Dispute Resolution and Moderation
+
+Disputes are inevitable, including violations of the rules above. When these occur, the following policy applies.
+
+- Before any other resolution step, we trust the community to police itself. If you see a community member violating these community standards, please let them know, and link them to this document. Do not respond in kind (e.g. respond to a flame with a flame). Most disagreements can be solved with education and discussion. If you have been linked to this document by another user, do not dwell on this point: alter your behavior as appropriate and move on.
+
+- If the issue cannot be resolved between the contributors, any complaints, instances of explicit rule breaking, or unresolvable disagreements with other community members may be directed towards the Staff or the Project Leader directly. You may do so through email (`team [at] openbullet.dev`). Please include details and context as appropriate. See the **(insert a link to staff members list and contact)** for the list of Staff members and ways to contact them.
```

Comment:
waiting for something I can do to find a way to contact the staff or Ruri

Reply:
Forum staff can be found at https://discourse.openbullet.dev/u?group=staff&order=likes_received
My email can be found at the bottom of [this readme](https://github.com/openbullet/OpenBullet2)

Reply:
Suggestions applied !